fix: HUMSIG-37: Visual fixes for Project settings

makseq opened the pull request:
## Annotation tab updated
![image](https://github.com/HumanSignal/label-studio/assets/501780/f7bd77c2-9d36-4d32-b14a-f06a8de353db)

## Annotation tab old
![image](https://github.com/HumanSignal/label-studio/assets/501780/161694c9-3ca4-4867-a5b5-800110a43b95)

------

## Model tab new
![image](https://github.com/HumanSignal/label-studio/assets/501780/75b05f70-f9f9-4532-aeda-a4e573e7a52c)

## Model tab old
![image](https://github.com/HumanSignal/label-studio/assets/501780/4e7af2c5-c95d-4bd4-a3a0-b0382f2caf55)

------

## Predictions tab new
![image](https://github.com/HumanSignal/label-studio/assets/501780/e4f81d78-e5a5-4d62-96ff-e97ff3d3f591)

## Predictions tab old
![image](https://github.com/HumanSignal/label-studio/assets/501780/8a85048e-fd5a-4e79-b547-1a74a58e4861)



### PR fulfills these requirements
- [ ] Tests for the changes have been added/updated
- [ ] Docs have been added/updated
- [ ] Best efforts were made to ensure docs/code are concise and coherent (checked for spelling/grammatical errors, commented out code, debug logs etc.)
- [x] Self-reviewed and ran all changes on a local instance

### Describe the reason for change
The previous interface had visual flaws. 

### What feature flags were used to cover this change?
None

### This change affects (describe how if yes)
- [ ] Performance
- [ ] Security
- [x] UX

### Does this PR introduce a breaking change?
- [ ] Yes, and covered entirely by feature flag(s)
- [ ] Yes, and covered partially by feature flag(s)
- [x] No
- [ ] Not sure (briefly explain the situation below)

makseq committed:
fix: HUMSIG-37: Visual fixes

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/8533471363

makseq commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/8540366352)
> Successfully merged: Already up to date.

hlomzik commented on web/apps/labelstudio/src/pages/Settings/AnnotationSettings/ModelVersionSelector.js:
```diff
@@ -72,13 +72,10 @@ export const ModelVersionSelector = ({
-
+    <div>
+      <p>
+          Select which predictions or which model you want to use:
```
wrong indent
```suggestion
        Select which predictions or which model you want to use:
```

hlomzik commented on web/apps/labelstudio/src/pages/Settings/MachineLearningSettings/MachineLearningList.js:
```diff
@@ -1,11 +1,12 @@
 import { Button, Dropdown, Menu } from '../../../components';
 import { confirm } from '../../../components/Modal/Modal';
 import { Oneof } from '../../../components/Oneof/Oneof';
+import {Tooltip} from "../../../components/Tooltip/Tooltip";
```
```suggestion
import { Tooltip } from "../../../components/Tooltip/Tooltip";
```

hlomzik commented on web/apps/labelstudio/src/pages/Settings/MachineLearningSettings/MachineLearningSettings.js:
```diff
@@ -129,20 +129,19 @@ export const MachineLearningSettings = () => {
 
         {backends.length > 0 && (
-          <Description style={{ marginTop: 0, maxWidth: 680 }}>
+          <Description style={{ marginTop: 0, maxWidth: 680, lineHeight:'1.5em' }}>
```
```suggestion
          <Description style={{ marginTop: 0, maxWidth: 680, lineHeight: "1.5em" }}>
```

hlomzik commented on web/apps/labelstudio/src/pages/Settings/PredictionsSettings/PredictionsSettings.js:
```diff
@@ -43,26 +44,19 @@ export const PredictionsSettings = () => {
-            <Caption>
-              Each card is associated with separate model version.
-            </Caption>
+            <Description  style={{ marginTop: 14 }}>
```
move this to styles file

hlomzik replied:
and why do you have description inside `title-block`?

hlomzik replied:
ah, that's external component, then this style is good.
but title-block > description is still a little confusing

hlomzik commented on web/apps/labelstudio/src/pages/Settings/PredictionsSettings/PredictionsSettings.js:
```diff
@@ -43,26 +44,19 @@ export const PredictionsSettings = () => {
-        )}
-
         {loaded && versions.length > 0 && (
           <Elem name={'title-block'}>
             <Elem name={'title'}>Predictions List</Elem>
```
```suggestion
          <Elem name="title-block">
            <Elem name="title">Predictions List</Elem>
```

hlomzik reviewed (changes requested):
minor comments

makseq committed:
Update web/apps/labelstudio/src/pages/Settings/AnnotationSettings/ModelVersionSelector.js

Co-authored-by: hlomzik <hlomzik@gmail.com>

makseq committed:
Update web/apps/labelstudio/src/pages/Settings/MachineLearningSettings/MachineLearningList.js

Co-authored-by: hlomzik <hlomzik@gmail.com>

makseq committed:
Update web/apps/labelstudio/src/pages/Settings/MachineLearningSettings/MachineLearningSettings.js

Co-authored-by: hlomzik <hlomzik@gmail.com>

makseq committed:
Update web/apps/labelstudio/src/pages/Settings/PredictionsSettings/PredictionsSettings.js

Co-authored-by: hlomzik <hlomzik@gmail.com>

makseq committed:
Fixes

makseq commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/8554694192)
> Successfully merged:  32 files changed, 298 insertions(+), 1030 deletions(-)

makseq committed:
Merge branch 'develop' into 'fb-HUMSIG-37-visuals'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/8554694192

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/8554722295

makseq committed:
Fixes

makseq committed:
Revert

makseq committed:
Fix eslintrc

makseq committed:
F

makseq committed:
Merge remote-tracking branch 'origin/fb-HUMSIG-37-visuals' into fb-HUMSIG-37-visuals

makseq committed:
Build

makseq committed:
Fix storages

makseq committed:
Annotation tab

makseq commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/8558118488)
> Successfully merged:  1 file changed, 4 insertions(+), 2 deletions(-)

makseq committed:
Merge branch 'develop' into 'fb-HUMSIG-37-visuals'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/8558118488

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/8558143196
